development/docker-compose.base.yml

```diff
@@ -23,6 +23,9 @@ services:
       - "redis"
     <<: *nautobot-build
     <<: *nautobot-base
+    # Uncomment if using Bot Framework Emulator
```

Comment:
I think this should be an additional `docker-compose` file to prevent an accidental commit of this being commented out.

Reply:
okay, thanks for the feedback.